example/components/list-example.vue

```diff
@@ -8,16 +8,16 @@
 
         <h4>List with item template</h4>
         <dx-list :items="listData">
-            <div slot="item" slot-scope="{ data, index }">
-                {{index + 1}} - <i>{{data.day}}</i>
-            </div>
-            <div slot="weekend" slot-scope="{ data, index }">
-                {{index + 1}} - <b>{{data.day}}</b>
-            </div>
+            <template #item="{ data, index }">
```

Comment:
I suggest keeping the previous example as well so that we have scoped templates snippets for both old and new approach